### DISCUSSION TITLE
How do you run it locally with Docker Compose Up and a postgres db?

### DISCUSSION THREAD

**Discussion by mxn2020** [langflow-ai/langflow#1791] [General]
I am struggling to get an instance to run locally using docker compose up.

I would also like to set it up with a postgres db so I can connect it to supabase. But this is the lesser isue now.

I get this log. Somehow langflow backend seems to shutdown immediately, while the frontend runs.

backend-1   | INFO:     Will watch for changes in these directories: ['/app']
backend-1   | INFO:     Uvicorn running on http://0.0.0.0:7860 (Press CTRL+C to quit)
backend-1   | INFO:     Started reloader process [1] using WatchFiles
frontend-1  | 
frontend-1  | > langflow@0.1.2 dev:docker
frontend-1  | > vite --host 0.0.0.0
frontend-1  | 
frontend-1  | 
frontend-1  |   VITE v4.5.3  ready in 608 ms
frontend-1  | 
frontend-1  |   ➜  Local:   http://localhost:3000/
frontend-1  |   ➜  Network: http://172.18.0.3:3000/
backend-1   | Starting Langflow v1.0.0a27...
backend-1   | Shutting down Langflow...
frontend-1  | 1:50:03 PM [vite] http proxy error at /health:
frontend-1  | Error: connect ECONNREFUSED 172.18.0.2:7860
frontend-1  |     at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1605:16)

**Comment by flefevre** [comment]
Could you share your docker-compose.yml file and its associated Dockerfile ?
Thanks

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
